specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -2340,6 +2402,39 @@
           "type": "string"
         }
       }
+    },
+    "AcsMessageLanguageDetection": {
+      "description": "Message Analysis Language Detection",
+      "type": "object",
+      "properties": {
+        "language": {
+          "description": "The language of the message",
+          "type": "string"
+        },
+        "confidenceScore": {
+          "description": "The confidence score of the language detected",
+          "type": "string"
```

Comment:
Why is this being defined as a string? The example shows a number.

Reply:
I think its because it can either long or int format

Reply:
The example does not show a string. Please investigate whether it should be a string or a number.

Reply:
its a string

Reply:
also this will be something better answered/discussed on the review day

Reply:
If it is meant to be a string, can you please update the example?